doc/classes/RenderingServer.xml

```diff
@@ -4239,6 +4239,9 @@
 		<constant name="MAX_2D_DIRECTIONAL_LIGHTS" value="8">
 			The maximum number of directional lights that can be rendered at a given time in 2D.
 		</constant>
+		<constant name="MAX_MESH_SURFACES" value="256">
+			Maximum number of surfaces a [Mesh] can have.
```

Comment:
```suggestion
			The maximum number of surfaces a mesh can have.
```

- Keep the format the same as other `MAX_*` constants.
- The `Mesh` resource is built on top of `RenderingServer`. It's just a general concept "mesh" here (created by `mesh_create()`).

Reply:
Done! Thanks for the adjust suggestion